src/components/Container/Container.vue

```diff
@@ -28,21 +28,21 @@ export default {
   methods: {
 
     formatTemp: function (location) {
-      const temperature = location.data.current.weather.tp
+      const temperature = location.current.weather.tp
       return (temperature * 9 / 5 + 32)
     },
 
     formatLocation: function (location) {
-      return `${location.data.city}, ${location.data.state}`
+      return `${location.city}, ${location.state}`
     },
 
     formatWindSpeed: function (location) {
-      const windSpeed = location.data.current.weather.ws
+      const windSpeed = location.current.weather.ws
       return parseInt((windSpeed * 2.237).toFixed(2))
     },
 
     formatTimeStamp: function (location) {
-      const timeStamp = location.data.current.weather.ts
+      const timeStamp = location.current.weather.ts
       const date = new Date(timeStamp)
       return (date.toDateString() + ' ' + date.toLocaleTimeString())
     }
```

Comment:
Tashia already made an issue / card for this - we'll want to return to refactor this formatting for the date and time. I think these can likely still live within the same function here, but will just need to be formatted and assigned to variables before the return.